queue_services/entity-filer/src/entity_filer/filing_processors/special_resolution.py

```diff
@@ -31,7 +31,7 @@ def process(business: Business, filing: Dict, filing_rec: Filing):
             party = Party(
                 first_name=signatory.get('givenName', '').upper(),
                 last_name=signatory.get('familyName', '').upper(),
-                middle_initial=signatory.get('additionalName', '').upper()
+                middle_initial=(signatory.get('additionalName', '') or '').upper()
```

Comment:
Optional field, fails when frontend provides `additionalName: None`

Reply:
Doesn't the `get` already return '' if the field isn't found?